app/web/endpoints/sheet.py

```diff
@@ -12,70 +13,117 @@
+    status_code=HTTPStatus.CREATED,
+    summary="Store a new Google Sheet for regular archiving.",
+)
 def create_sheet(
     sheet: schemas.SheetAdd,
     user: UserState = Depends(get_user_state),
     db: Session = Depends(get_db_dependency),
 ) -> schemas.SheetResponse:
-
     if not user.in_group(sheet.group_id):
-        raise HTTPException(status_code=403, detail="User does not have access to this group.")
+        raise HTTPException(
+            status_code=HTTPStatus.FORBIDDEN,
+            detail="User does not have access to this group.",
+        )
 
     if not user.has_quota_monthly_sheets(sheet.group_id):
-        raise HTTPException(status_code=429, detail="User has reached their sheet quota for this group.")
+        raise HTTPException(
+            status_code=HTTPStatus.TOO_MANY_REQUESTS,
+            detail="User has reached their sheet quota for this group.",
+        )
 
     if not user.is_sheet_frequency_allowed(sheet.group_id, sheet.frequency):
-        raise HTTPException(status_code=422, detail="Invalid frequency selected for this group.")
+        raise HTTPException(
+            status_code=HTTPStatus.UNPROCESSABLE_ENTITY,
+            detail="Invalid frequency selected for this group.",
+        )
 
     try:
-        return crud.create_sheet(db, sheet.id, sheet.name, user.email, sheet.group_id, sheet.frequency)
+        return crud.create_sheet(
+            db,
+            sheet.id,
+            sheet.name,
+            user.email,
+            sheet.group_id,
+            sheet.frequency,
+        )
     except exc.IntegrityError as e:
-        raise HTTPException(status_code=400, detail="Sheet with this ID is already being archived.") from e
+        raise HTTPException(
+            status_code=HTTPStatus.BAD_REQUEST,
+            detail="Sheet with this ID is already being archived.",
+        ) from e
 
 
-@sheet_router.get("/mine", status_code=200, summary="Get the authenticated user's Google Sheets.")
+@sheet_router.get(
+    "/mine",
+    status_code=HTTPStatus.OK,
+    summary="Get the authenticated user's Google Sheets.",
+)
 def get_user_sheets(
     user: UserState = Depends(get_user_state),
-    db: Session = Depends(get_db_dependency)
+    db: Session = Depends(get_db_dependency),
 ) -> list[schemas.SheetResponse]:
     return crud.get_user_sheets(db, user.email)
 
 
-@sheet_router.delete("/{id}", summary="Delete a Google Sheet by ID.")
+@sheet_router.delete("/{sheet_id}", summary="Delete a Google Sheet by ID.")
 def delete_sheet(
-    id: str,
+    sheet_id: str,
     user: UserState = Depends(get_user_state),
     db: Session = Depends(get_db_dependency),
-) -> schemas.DeleteResponse:
-    return JSONResponse({
-        "id": id,
-        "deleted": crud.delete_sheet(db, id, user.email)
-    })
-
-
-@sheet_router.post("/{id}/archive", status_code=201, summary="Trigger an archiving task for a GSheet you own.", response_description="task_id for the archiving task.")
+) -> JSONResponse:
```

Comment:
DeleteResponse seems better for /docs interpretation

Reply:
That's fine, but the `DeleteResponse` object doesn't match what's actually returned from the function:
```python
return JSONResponse(
        {"id": sheet_id, "deleted": crud.delete_sheet(db, sheet_id, user.email)}
    )
```

Reply:
Done.